src/components/Provider.svelte

```diff
@@ -1,61 +1,84 @@
 <script lang="ts">
   import {
     dotApi,
-    storeProviderId,
     storeConnected,
+    storeCurrentAction,
+    storeMsaInfo,
     storeToken,
     transactionSigningAddress
   } from '$lib/stores';
-  import {ActionForms} from '$lib/storeTypes';
-  import type { AccountInfo} from "$lib/storeTypes";
-  import { storeCurrentAction } from '$lib/stores.js';
+  import type {MsaInfo} from '$lib/storeTypes';
+  import {ActionForms} from "$lib/storeTypes";
   import type {ApiPromise} from "@polkadot/api";
-  import { formatBalance } from "@polkadot/util";
-  import { getBalances} from "$lib/polkadotApi";
+  import {formatBalance} from "@polkadot/util";
+  import {getBalances} from "$lib/polkadotApi";
+  import type { AccountBalances } from "$lib/polkadotApi";
+  import { isMainnet } from "$lib/utils";
 
-  // the locally stored value of the provider Id
-  let localProvider = 0;
-  storeProviderId.subscribe((val) => (localProvider = val));
   let connected = false;
   storeConnected.subscribe((val) => (connected = val));
 
-  let token = '';
-  storeToken.subscribe(val => token = val);
-
   let api: ApiPromise;
+  let network:string  = '';
   dotApi.subscribe(storeDotApi => {
+    network = storeDotApi?.selectedEndpoint || '';
     if (storeConnected && storeDotApi.api) {
       api = storeDotApi.api
     }
   })
 
-  let accountInfo: AccountInfo = { balanceFree: 0n, balanceReserved: 0n, balanceFrozen: 0n, balanceTotal: 0n};
+  let token = '';
+  storeToken.subscribe(val => token = val.toString());
+
   let localSigningAddress = '';
   transactionSigningAddress.subscribe(async val => {
     localSigningAddress = val;
     if (api) {
-      accountInfo = await getBalances(api, val);
+      accountBalances = await getBalances(api, val);
     }
   });
 
+  // the locally stored value of the provider Id
+  let msaId = 0;
+  let isProvider = false;
+
+  storeMsaInfo.subscribe((info: MsaInfo) => {
+    msaId = info?.msaId || 0;
+    isProvider = info?.isProvider || false
+  });
+
+  let accountBalances: AccountBalances = { free: 0n, reserved: 0n, frozen: 0n};
+
   const balanceToHuman = (balance: bigint): string => {
     return formatBalance(balance, {withSiFull: true, withUnit: token, withZero: true})
```

Comment:
Not this PR, but it is missing `decimals` (it defaults to 0: https://github.com/polkadot-js/common/blob/master/packages/util/src/format/formatBalance.ts#L67)

Reply:
I'm not what you mean. The output format defaults to 4 and despite trying, I couldn't make it any different just using the polkadotjs formatter:
![Screenshot 2023-08-11 at 11 09 38 AM](https://github.com/LibertyDSNP/provider-dashboard/assets/502640/99a9ba32-7b40-40b9-a2bd-81c688406bdb)

Reply:
You don't have a Trillion XRQCY however right?

When I do it on testnet with the 500 XRQCY, it then reports 500 Bill when it should be `500 "" XRQCY`

<img width="257" alt="image" src="https://github.com/LibertyDSNP/provider-dashboard/assets/1252199/a385efbf-063f-481c-9ecf-fd9cd1539159">